modules/apps/transfer/types/v3/packet_test.go

```diff
@@ -0,0 +1,571 @@
+package v3
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cometbft/cometbft/crypto/secp256k1"
+
+	"github.com/cosmos/ibc-go/v8/modules/apps/transfer/types"
+	ibcerrors "github.com/cosmos/ibc-go/v8/modules/core/errors"
+)
+
+const (
+	denom              = "transfer/gaiachannel/atom"
+	amount             = "100"
```

Comment:
I think these values are not used in the tests, you could set them with the values you currently use (`atom/pool` and `1000`).